tests/test_jexl.py

```diff
@@ -118,6 +121,34 @@ def test_validate():
     assert errors == ['Could not parse expression: "\n"']
 
 
+JEXL_ALPHABET = strategies.characters(whitelist_categories=(
+    # Letters
+    'Lu', 'Ll',
+    # Numbers
+    'Nd', 'No',
+    # Spaces
+    'Zs', 'Zl', 'Zp',
+), whitelist_characters=[
+    '\t', '\n', '\r', ' ',
+    '$', '|', '+', '/',
+    '(', ')', '{', '}', '[', ']',
+    "'", '"', ',', ':', '?',
+])
+
+
+@hypothesis.given(strategies.text(JEXL_ALPHABET))
+@hypothesis.settings(max_examples=500)
+def test_validate_always_returns(s):
+    jexl = JEXL()
+    errors = list(jexl.validate(s))
+    assert isinstance(errors, list)
+
+    # I couldn't figure out how to generate '{{' and '}}' (or, indeed,
+    # any general sequence of characters), so just generate it intentionally.
+    errors = list(jexl.validate(s.replace('{', '{{').replace('}', '}}')))
+    assert isinstance(errors, list)
```

Comment:
I'm guessing the whole point of this is to make sure it doesn't crash at all. Is that right? If so, a comment about that would be awesome.